code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -132,6 +132,15 @@
 		. = 1
 	..()
 
+/datum/reagent/toxin/shellblood
```

Comment:
If this is still basically 'blood', maybe it should add a little as it metabolizes?